apiv2/schema/schema.yaml

```diff
@@ -3,7 +3,7 @@ name: cdp-dataset-config
 description: Schema for dataset configs
 version: 1.1.0
 imports:
-- linkml:types
```

Comment:
my new editor config wants to indent lists now, and the community is waffly about which behavior is correct -- https://github.com/yaml/pyyaml/issues/234 😭